doc/index.rst

```diff
@@ -12,10 +12,10 @@ A high-level app and dashboarding solution for Python
    <table>
      <tr>
        <td border=1><a href="https://examples.pyviz.org/attractors/clifford_panel.html"><b>Interact</b></a><br><a href="https://clifford.pyviz.demo.anaconda.com"><img src="_images/clifford.png" /></a></td>
-       <td border=1><a href="https://examples.pyviz.org/gapminders/gapminders.html"><b>Gapminders</b></a><br><a href="https://gapminder.pyviz.demo.anaconda.com"><img src="_images/gapminders.png" /></a></td>
+       <td border=1><a href="https://examples.pyviz.org/gapminders/gapminders.html"><b>Gapminders</b></a><br><a href="https://gapminder.pyviz.demo.anaconda.com"><img src="_images/gapminder.png" /></a></td>
```

Comment:
The actual app is called "gapminders" with an "s", so it's odd that the thumbnail would be called "gapminder"; maybe rename the file instead?